app/connectors/SecureCommsAlertConnector.scala

```diff
@@ -40,7 +40,6 @@ class SecureCommsAlertConnector @Inject()(httpClient: HttpClient,
       response.status match {
         case OK => handleOk(response)
         case BAD_REQUEST => handleBadRequest(response)
-        case NOT_FOUND => handleNotFound(response)
```

Comment:
The change needs to be made in `SecureMessageService`. There are two "not found" errors there with status code 404 that need to be removed. This connector and the related service that was also changed should not be changed as they will still need to cater for 404 responses.

I appreciate some of the naming conventions in this service are very confusing 😅